test/mongodb.test.js

```diff
@@ -1611,6 +1612,67 @@ describe('mongodb connector', function() {
       });
     });
 
+    it('find should be able to query around location in deep/multiple keys', function(done) {
+      var coords = { lat: 1.25, lng: 20.20 };
+
+      geoDb.autoupdate(function(err) {
+        var postNb = 0;
+
+        var createPostWithLocation = function(callback) {
+          postNb++;
+
+          PostWithLocation.create({
+            title: 'Post #' + postNb,
+            location: new GeoPoint({
+              lat: coords.lat + (Math.random() * 2 - 1),
```

Comment:
I removed the random value. Fixed coordinates shouldn't affect the behavior of the test so much.